python/nano/example/pytorch/inference_pipeline/vision_transformers/vision_transformer.ipynb

```diff
@@ -0,0 +1,795 @@
+    "\n",
+    "optimizer = InferenceOptimizer()\n",
+    "optimizer.optimize(model,\n",
+    "                   training_data=faked_train_dataloader,\n",
+    "                   validation_data=val_dataloader,\n",
+    "                   metric=Accuracy(),\n",
+    "                   direction=\"max\",\n",
+    "                   thread_num=8)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " -------------------------------- ---------------------- -------------- ----------------------\n",
+      "|             method             |        status        | latency(ms)  |       accuracy       |\n",
+      " -------------------------------- ---------------------- -------------- ----------------------\n",
+      "|            original            |      successful      |    96.791    |         0.8          |\n",
+      "|           fp32_ipex            |      successful      |    95.871    |    not recomputed    |\n",
+      "|              bf16              |   fail to forward    |     None     |         None         |\n",
+      "|           bf16_ipex            |    early stopped     |   903.938    |         None         |\n",
+      "|              int8              |      successful      |    86.094    |         0.85         |\n",
+      "|            jit_fp32            |      successful      |    81.166    |    not recomputed    |\n",
+      "|         jit_fp32_ipex          |      successful      |     80.6     |    not recomputed    |\n",
+      "|  jit_fp32_ipex_channels_last   |      successful      |    76.594    |    not recomputed    |\n",
+      "|         openvino_fp32          |      successful      |    37.541    |    not recomputed    |\n",
+      "|         openvino_int8          |      successful      |   181.284    |         0.0          |\n",
+      "|        onnxruntime_fp32        |      successful      |    91.667    |    not recomputed    |\n",
+      "|    onnxruntime_int8_qlinear    |      successful      |    70.139    |         0.6          |\n",
+      "|    onnxruntime_int8_integer    |   fail to convert    |     None     |         None         |\n",
+      " -------------------------------- ---------------------- -------------- ----------------------\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "optimizer.summary()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# step4 : save and load model (optional)\n",
+    "After you get an accelerated model, you can save it as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "InferenceOptimizer.save(acc_model, path=\"ckpt\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then load it by `Trainer.load`"
```

Comment:
Then load it by `InferenceOptimizer.load`

We need to show case how the user can export the optimized model in standard format (PyTorch, ONNX, OpenVINO, etc.)

Reply:
` InferenceOptimizer.load` -> will fix this bug in next PR.

`how the user can export the optimized model in standard format (PyTorch, ONNX, OpenVINO)`  this case should be put in this example or another independent example ?

Reply:
Maybe another example?